.vscode/tasks.json

```diff
@@ -0,0 +1,19 @@
+{
+	"version": "2.0.0",
+	"tasks": [
+        {
+            "label": "rust: cargo build (win32)",
+            "args": [
+                "build",
+                "--release",
+                "--target",
+                "i686-pc-windows-msvc"
```

Comment:
Should these be merged since its the same flag?

Reply:
100%

Reply:
The task will not run if they are merged.
```
error: unexpected argument '--target i686-pc-windows-msvc' found

  tip: a similar argument exists: '--target'

Usage: cargo.exe build --release --target <TRIPLE>
```